File: 20.06.2025/TemperatureConverter.java
public class TemperatureConverter {
    public static double celsiusToFahrenheit(double celsius) {
        return (celsius * 9 / 5) + 32;
    }

    public static double fahrenheitToCelsius(double fahrenheit) {
        return (fahrenheit - 32) * 5 / 9;
    }

    public static void main(String[] args) {
        System.out.println("0 Celsius is " + celsiusToFahrenheit(0) + " Fahrenheit.");
        System.out.println("32 Fahrenheit is " + fahrenheitToCelsius(32) + " Celsius.");
    }
}

File: 29.06.2025/temperature_converter.py
# Filename: temperature_converter.py
class TemperatureConverter:
    def __init__(self, celsius=None, fahrenheit=None):
        self.celsius = celsius
        self.fahrenheit = fahrenheit

    def convert_celsius_to_fahrenheit(self):
        if self.celsius is not None:
            return (self.celsius * 9/5) + 32
        return None

    def convert_fahrenheit_to_celsius(self):
        if self.fahrenheit is not None:
            return (self.fahrenheit - 32) * 5/9
        return None